DeployClient.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+CLIENT=$1
+GEN_TYPE=$2
+
+# echo "GEN_TYPE=${GEN_TYPE}, CLIENT=${CLIENT}"
+
+echo "Build client"
+cd $CLIENT
+yarn install
+yarn lint
+if [[ $GEN_TYPE != None ]] 
```

Comment:
What is `None` here? That's not a bash thing, is it?
Also I'm not sure what the point of this test here.
Instead of GEN_TYPE don't you want GENERATE y/n??? Why "type"?
```suggestion
if [[ $GEN_TYPE != None ]] 
```

Reply:
@hillairet This is something we do [sb-pim](https://github.com/SatelCreative/sb-pim/blob/d8141158f4d2321173d276b4aba166c4dde8b5b4/Jenkinsfile#L25) related to graphql I believe. I set the default value to `None` in [actions.yml](https://github.com/SatelCreative/satel-client-deployment/blob/a1e253395632e03d316a3283f47973a0083443b9/action.yml#L11) for the project we dont use it like `st-pim`, but that can be change to something like `if [[ $GEN_TYPE != True ]]`

Reply:
Well my point is that at least you should use `-n` or `-z` to check if the variable is empty. `None` is a python thing!
Also use `true` and not `True`

Reply:
Yeah I changed it to use `-n` after  I posted this comment.